src/dotnet/commands/RestoringCommand.cs

```diff
@@ -12,31 +12,36 @@ public class RestoringCommand : MSBuildForwardingApp
     {
         private bool NoRestore { get; }
 
-        private IEnumerable<string> ArgsToForward { get; }
+        private IEnumerable<string> UserDefinedArguments { get; }
```

Comment:
I'm having trouble understanding the difference between Trailing and UserDefined. In particular, what I call trailing could also be user-defined in the sense that the user is passing options that are outside the verb definition.